.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Bug report
+about: Create a report to help us improve
```

Comment:
"Let us know when something isn't working correctly"?

"Tell us about a problem you're experiencing"?

Reply:
I reworded this one.